client/asset/dcr/wallet.go

```diff
@@ -92,54 +86,64 @@ type Wallet interface {
+	// returned block header is a wire.BlockHeader with the addition of the block's
+	// median time.
+	GetBlockHeader(ctx context.Context, blockHash *chainhash.Hash) (*BlockHeader, error)
+	// GetBlock returns the *wire.MsgBlock.
+	GetBlock(ctx context.Context, blockHash *chainhash.Hash) (*wire.MsgBlock, error)
 	// GetTransaction returns the details of a wallet tx, if the wallet contains a
 	// tx with the provided hash. Returns asset.CoinNotFoundError if the tx is not
 	// found in the wallet.
 	GetTransaction(ctx context.Context, txHash *chainhash.Hash) (*walletjson.GetTransactionResult, error)
-	// GetRawTransactionVerbose returns details of the tx with the provided hash.
+	// GetRawTransaction returns details of the tx with the provided hash.
 	// Returns asset.CoinNotFoundError if the tx is not found.
-	GetRawTransactionVerbose(ctx context.Context, txHash *chainhash.Hash) (*chainjson.TxRawResult, error)
-	// GetRawMempool returns hashes for all txs of the specified type in the node's
-	// mempool.
-	GetRawMempool(ctx context.Context, txType chainjson.GetRawMempoolTxTypeCmd) ([]*chainhash.Hash, error)
+	GetRawTransaction(ctx context.Context, txHash *chainhash.Hash) (*wire.MsgTx, error)
 	// GetBestBlock returns the hash and height of the wallet's best block.
 	GetBestBlock(ctx context.Context) (*chainhash.Hash, int64, error)
 	// GetBlockHash returns the hash of the mainchain block at the specified height.
 	GetBlockHash(ctx context.Context, blockHeight int64) (*chainhash.Hash, error)
-	// BlockCFilter fetches the block filter info for the specified block.
-	BlockCFilter(ctx context.Context, blockHash *chainhash.Hash) (filter, key string, err error)
-	// LockWallet locks the wallet.
-	LockWallet(ctx context.Context) error
-	// UnlockWallet unlocks the wallet.
-	UnlockWallet(ctx context.Context, passphrase string, timeoutSecs int64) error
-	// WalletUnlocked returns true if the wallet is unlocked.
-	WalletUnlocked(ctx context.Context) bool
-	// AccountUnlocked returns true if the specified account is unlocked.
-	AccountUnlocked(ctx context.Context, account string) (*walletjson.AccountUnlockedResult, error)
-	// LockAccount locks the specified account.
-	LockAccount(ctx context.Context, account string) error
-	// UnlockAccount unlocks the specified account.
-	UnlockAccount(ctx context.Context, account, passphrase string) error
+	// BlockFilter fetches the block filter info for the specified block.
+	BlockFilter(ctx context.Context, blockHash *chainhash.Hash) ([gcs.KeySize]byte, *gcs.FilterV2, error)
+	// AccountUnlocked returns true if the account is unlocked.
+	AccountUnlocked(ctx context.Context, acctName string) (bool, error)
+	// LockAccount locks the account.
+	LockAccount(ctx context.Context, acctName string) error
+	// UnlockAccount unlocks the Wallet. ExchangeWallet does not differentiate
+	// account locking vs wallet locking, but the underlying implementation may
+	// choose to unlock only the account.
```

Comment:
Should this explanation also apply to `AccountUnlocked` and `LockAccount`?

Reply:
Just needs to be removed, I guess. Applied to previous revision with method `Unlock`.